spec/controllers/user_sessions_controller_spec.rb

```diff
@@ -1,15 +1,18 @@
 require 'rails_helper'
 
 RSpec.describe UserSessionsController, type: :controller do
+  before(:each) do
+    @user = create(:user, password: 'valid')
+    @user.activate!
+  end
+
   it 'logs user in when valid' do
-    user = create(:user, password: 'valid')
-    post :create, session: { email: user.email, password: 'valid' }
-    expect(response).to redirect_to '/user_groups'
+    post :create, session: { email: @user.email, password: 'valid' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.